src/patterns/components/card/react/code/teaser.hbs

```diff
@@ -0,0 +1,17 @@
+<SprkCard idString="card1" teaserConfig={teaserConfig} />
+
+<script>
+  const teaserConfig = {
+    bodyText: 'Body text of a teaser card.',
```

Comment:
Values here need to match the docs.

Reply:
Hey @RVMendoza , I saw that you marked this as resolved but didn't make a change or comment. What are your thoughts?